physics/bernouli_principle.py

```diff
@@ -0,0 +1,25 @@
+# First, make sure you have Matplotlib installed. You can install it using pip if you haven't already
```

Comment:
make sure pass through ruff

Reply:
what changes should I do can please suggest me

Reply:
please see the ruff, pre-commit failures